README.md

```diff
@@ -11,6 +11,10 @@ Seriously, we need and want the help. Want to learn how to code? Or maybe you kn
 autoscend is a script that will play through an entire ascension for you in the Kingdom of Loathing.
 It is built up from sl\_ascend and cc\_ascend before it.
 
+autoscends goal is supposed to be able to brute force an ascension for all paths of KoL, Not to do it optimally, but within a couple real-life days of target. If you bear this in mind while you use it you will have a much better experience.  
+If however, you expect it to run absolutely bleeding edge ascensions with optimal day/turn counts you are in for a world of disappointment.  
+A general purpose ascension script, which supports accounts with any number of perm's and Mr Store items from none to almost all, will not be optimally tuned to your specific situation. We suggest you adjust your expectations accordingly.
```

Comment:
A general purpose ascension script, which supports accounts with any number of perms and/or Mr Store items from none to almost all, &c &c